com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -51,32 +78,55 @@ public enum ShaderOptions
         // Edit->Render Pipeline->Generate Shader Includes
         // Probe Volumes feature must also be enabled inside of your HDRenderPipelineAsset.
         // Also uncomment in the HDRP package all ".../Experimental/Probe Volume" menu
+
+        /// <summary>Probe volume evaluation mode.</summary>
         ProbeVolumesEvaluationMode = ProbeVolumesEvaluationModes.Disabled,
+        /// <summary>Probe volume supports additive blending.</summary>
         ProbeVolumesAdditiveBlending = 1,
+        /// <summary>Probe volume filtering mode.</summary>
         ProbeVolumesBilateralFilteringMode = ProbeVolumesBilateralFilteringModes.Validity,
+        /// <summary>Probe volume encoding.</summary>
         ProbeVolumesEncodingMode = ProbeVolumesEncodingModes.SphericalHarmonicsL1,
 
+        /// <summary>Support for area lights.</summary>
         AreaLights = 1,
 
+        /// <summary>Support for barn doors.</summary>
         BarnDoor = 0
     };
 
     // Note: #define can't be use in include file in C# so we chose this way to configure both C# and hlsl
     // Changing a value in this enum Config here require to regenerate the hlsl include and recompile C# and shaders
+    /// <summary>
+    /// Project wide shader configuration options.
+    /// This class reflects the ShaderOptions enum. It's meant to be used in C# code to check current configuration.
+    /// </summary>
     public class ShaderConfig
     {
-        public const int k_XRMaxViewsForCBuffer = 2;         // REALLY IMPORTANT! This needs to be the maximum possible XrMaxViews for any supported platform!
-                                                             // this needs to be constant and not vary like XrMaxViews does as it is used to generate the cbuffer declarations
+        // REALLY IMPORTANT! This needs to be the maximum possible XrMaxViews for any supported platform!
+        // this needs to be constant and not vary like XrMaxViews does as it is used to generate the cbuffer declarations
+        /// <summary>Maximum number of XR views for constant buffer allocation.</summary>
+        public const int k_XRMaxViewsForCBuffer = 2;
 
+        /// <summary>Use camera relative rendering to enhance precision.</summary>
```

Comment:
Indicates whether to use [camera-relative rendering](../manual/Camera-Relative-Rendering.md) to enhance precision. A value of 1 enables camera-relative rendering and a value of 0 disables camera-relative rendering.

Reply:
For all of these below properties, can you add:  A value of x enables _feature_ and a value of 0 disables _feature_. Make it a [list](https://confluence.unity3d.com/pages/viewpage.action?spaceKey=DOCS&title=API+docs+content+and+formatting#sphnav93901911Packages) if there is more than one value.

Reply:
Also can you add a ///\<seealso cref=""/> with a  link to the shader options value? Just so we don't have to duplicate information.